src/core/numerics/ohm/ohm.h

```diff
@@ -10,14 +10,29 @@
 #include "core/data/vecfield/vecfield_component.h"
 #include "core/utilities/index/index.h"
 
+#include "phare_core.h"
+#include "initializer/data_provider.h"
+
+
 namespace PHARE
 {
 namespace core
 {
     template<typename GridLayout>
     class Ohm : public LayoutHolder<GridLayout>
     {
+    public:
+        explicit Ohm(PHARE::initializer::PHAREDict dict)
+            : eta_{dict["resistivity"].template to<double>()}
+            , nu_{dict["hyper_resistivity"].template to<double>()}
+        {
+        }
+        // Ohm() = default;
```

Comment:
remove?

Reply:
not clear to why should we remove these 4 lines... init `nu_` is important no ?